src/Providers/LaravelInstallerServiceProvider.php

```diff
@@ -35,7 +37,13 @@ public function register()
      */
     public function boot(Router $router)
     {
-        $router->middlewareGroup('install',[CanInstall::class]);
+        if (empty(config('app.key'))) {
```

Comment:
is it possible to move this code [EnvironmentManager.php](https://github.com/ChargePanda-Organization/LaravelInstaller/pull/2/files#diff-d2602af967ed2ab1156dcd0883e0e2e788b10db8f33320943792725771275324) and recall this code right after copying the file?